charts/mysql/values.schema.json

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
I think we should remove this file. We have not done a good job of maintaining our own jsonschema file, and I doubt we would do a good job of maintaining this one.